railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -395,7 +395,6 @@ def delete_action_mailer_files_skipping_action_mailer
 
       def delete_action_cable_files_skipping_action_cable
         if options[:skip_action_cable]
-          remove_file "config/cable.yml"
           remove_file "app/assets/javascripts/cable.js"
           remove_dir "app/channels"
```

Comment:
Other files created by the method `create_app_files`. Generator copies `railties/lib/rails/generators/rails/app/templates/app/` directory  then gets rid of files in accordance with passed `options`.